app/screens/channel/channel_base.js

```diff
@@ -121,6 +122,10 @@ export default class ChannelBase extends PureComponent {
             this.loadChannels(nextProps.currentTeamId);
         }
 
+        if (nextProps.currentChannelId !== this.props.currentChannelId) {
+            this.props.actions.getChannelStats(nextProps.currentChannelId);
```

Comment:
why do we need the stats here?

Reply:
I added a new "guest_count" to the channel status, allowing the channels which contains guests to show that information directly in the header.

Reply:
Then why not make this call in the handleSelectChannel action instead?